### ISSUE TITLE
TC7200_Kernel_Image

### ISSUE THREAD

**Issue by jagmic** [jclehner/bcm2-utils#49]
Hello.
I'm looking for properly image kernel, app and rootfs for tc_7200. When I was trying to flash new image I put firmware to wrong place -. Now device is working , but linux not . I tried to compile myself form source TC72XX_LxG1.0.10mp5_OpenSrc , but code seems to be obsolete and all the time I have lot of errors. If you have these files - could you put it somwhere please ??

**Comment by jclehner** [comment]
Check out [this repository](https://github.com/jclehner/bcm2-dumps/tree/master/tc7200). The `LNXDC.01.01-*` files should work with the `TC7200.20-DC*` images.

**Comment by jagmic** [comment]
Yes . Thanks. Now is much better but not perfect .I mean - linux try to start but can't open ubi0:rootfs. how to load rootfs and linuxapp ? It is possible through bootloader ?
![obraz](https://user-images.githubusercontent.com/127594403/235372662-e289fc9a-aee5-4374-ad71-ea1ce11f7453.png)

**Comment by jclehner** [comment]
Sorry for the late reply, but have you flashed the `LNXDC.01.01-rootfs` and `-linuxapps` partitions as well?

**Comment by jagmic** [comment]
Hello . thx for reply. Regarding your question - yes I flashed all three parts - linux, rootfs and linuxapps. Main trouble appeared during attempt of flash rootfs and linuxapps because they are to big. Then I use ingore of bad crc "w 83F87630 0" . But no evail . My old linux kernel has different number LNXD6.02.07-kernel-20140224.bin . please take a look old log when device has worked 
[linux.txt](https://github.com/jclehner/bcm2-utils/files/11852418/linux.txt)

**Comment by jagmic** [comment]
Hello again. I'm looking firmware for tc7200.20 - it names STCF.01.42 ( full name TC7200-CF.01.42-161018-F-1c1.bin ) . I need to test some vpn connection and that sw consist additional tab for configure . do you have maybe this SW ? 
greetings
